script/check_copyright_date.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Check if the modified files' copyright dates are current.
+
+set -e
+
+# Change directory to top of repository.
+cd `dirname $0`
+cd ../
+
+# Get list of changed files.
+git fetch origin main
+modified_files=$(git diff --name-only --diff-filter=AM origin/main..HEAD)
+
+current_year="2024"
```

Comment:
is that possible to get the current year automatically?

Reply:
Yes, that's a good idea.